src/server/routes/queueDelete.ts

```diff
@@ -73,7 +72,11 @@ router.post(
 );
 
 /**
- * Enqueue item IDs for deletions in batches
+ * Enqueue item IDs for deletions in batches. This is used to purge
+ * user list and tag data when a user deletes their account. Since
+ * the data to clear could be large, we don't want to keep the api
+ * connection open while it's happening. Instead these processes
+ * will happen asynchronously in the background using queues.
```

Comment:
nice comment on async, will take this for account-data-deleter :D

Reply:
in account-data-deleter, changing this as `Instead the generation of the sqs messages will happen asynchronously in the background.`